_posts/2021-09-22-#22871.md

```diff
@@ -4,21 +4,109 @@ date: 2021-09-22
+      purposes in their nSequence numbers, as it would prohibit their transactions from propagating.
+
+* The PR author [posted to the mailing
+  list](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-September/019400.html)
+  soliciting feedback on this proposal.
+
+    - Dave Harding
+      [noted](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-September/019402.html) on the
+      mailing list that upgrades to relative timelock semantics would also be gated on a different nVersion number.
+
+    - Pieter Wuille [commented](https://github.com/bitcoin/bitcoin/pull/22871#issuecomment-915689161)
+      that, since nVersion numbers greater than 2 are discouraged in policy, this method of upgrading
+      relative timelocks is preserved without discouraging use of the locktime disable flag.
+
+    - Antoine Poinsot
+      [noted](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-September/019407.html)
+      on the mailing list that the Lightning Network protocol uses nSequence numbers to encode
+      commitment transaction numbers, including the disable locktime flag.
+
 ## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
 
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or
+   NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+
+2. Can you summarize the changes being proposed in this PR?
+
+3. What is the difference between policy and consensus rules?
+
+4. Why do we discourage the usage of upgradable NOPs in policy? How is this implemented? (Hint: grep
+   for `SCRIPT_VERIFY_DISCOURAGE_UPGRADABLE_NOPS` in src/script/interpreter.cpp).
+
+5. What are some reasons to discourage use of the locktime disable flag in policy?
+
+6. What are some reasons **not** to discourage use of the locktime disable flag in policy?
+
+7. What do you think of the
+   [comment](https://github.com/bitcoin/bitcoin/pull/22871#issuecomment-915689161) and
+   [response](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-September/019403.html) about
+   upgrading the relative timelock semantics by increasing the nVersion number?
+
+8. Do you think this change (discouragement of setting the most significant bit) should be applied
+   to nSequence numbers, CSV values, both, or neither? Why?
+
+10. What do you think of the approach of reusing the `SCRIPT_VERIFY_DISCOURAGE_UPGRADABLE_NOPS`
+   script verification flag to discourage use of the locktime disable flag?
+
+<!-- 9. Why is [this
+   commit](https://github.com/bitcoin-core-review-club/bitcoin/commit/e5b2a824147c342bfbf2c8b1696afc686c2bfda7),
+   which removes the `SCRIPT_VERIFY_DISCOURAGE_UPGRADABLE_NOPS` script verification flag from static
+   OP\_CSV tests, needed?
+
+   This commit is no longer part of the PR. TODO: update or remove this question. -->
```

Comment:
Note: it's a link to the bitcoin-core-review-club repo :stuck_out_tongue:

Reply:
oops. Now fixed. Thanks!